### ISSUE TITLE
go.owl says part_of symmetric. It is not. Probably meant reflexive.

### ISSUE THREAD

**Issue by alanruttenberg** [geneontology/go-ontology#12654]
Symmetric means if forall a,b, a part_of b => b part_of a. reflexive means forall a, a part_of a
See https://www.w3.org/TR/owl2-syntax/#Symmetric_Object_Properties
An example of a symmetric property would be "is beside". If a is beside b then b is beside a.

```
    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BFO_0000050">
        <owl:inverseOf rdf:resource="http://purl.obolibrary.org/obo/BFO_0000051"/>

here --> 
        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#SymmetricProperty"/>


        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#TransitiveProperty"/>
        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">BFO:0000050</oboInOwl:hasDbXref>
        <oboInOwl:hasOBONamespace rdf:datatype="http://www.w3.org/2001/XMLSchema#string">external</oboInOwl:hasOBONamespace>
        <oboInOwl:id rdf:datatype="http://www.w3.org/2001/XMLSchema#string">part_of</oboInOwl:id>
        <oboInOwl:shorthand rdf:datatype="http://www.w3.org/2001/XMLSchema#string">part_of</oboInOwl:shorthand>
        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">part of</rdfs:label>
    </owl:ObjectProperty>
```

**Comment by cmungall** [comment]
Wow, this is horrific.

I assume this leaked in via an import and is sticking due to the broken 
oboedit import mechanism.

**Comment by cmungall** [comment]
This crept in sometime in the last 2.5 months, don't have time to find 
out when.

Unfortunately, due to broken oboedit imports, if a bad OP axiom 
temporarily leaks into the import chain, if only for a day, it will 
persist in the main GO until manually removed. We'll have to set up a 
very specific check for things like this, there is no good general 
solution (other than ditching OE, which editors are not prepared to do. 
This will not be fixed in OE).

I fixed this for now in the editors file with r35496

If there is a go release tonight, it will be visible in public then.

I made an ad-hoc check in r35497
